### DISCUSSION TITLE
I cannot make the volume work with a LG 34UM94C-P monitor

### DISCUSSION THREAD

**Discussion by tsolignani** [MonitorControl/MonitorControl#1516] [Q&A]
Good afternoon.

The title says it all. Brightness works like a charme, but not the volume.

Tried playing with settings (not advanced) with no joy.

Hooked to a M2 Mini with USB to HDMI cable.

Thank you and have a nice day.

Tiziano

**Comment by vetomir** [comment]
same problem with LG OLED C3 42

**Comment by zhao233323** [comment]
Same problem on LG 32UN550-WAJP hooked on Mac mini with M2 chip by HDMI.

**Comment by nettanel** [comment]
Same here with DELL U2417H - slider move but the sound remains the same. 
Macbook pro m3.

**Comment by filiprd** [comment]
Same here with dual HP Z24m G3 monitors connected using DisplayPort to Macbook Pro with Intel chip.
